lib/widgets/organization_search_list.dart

```diff
@@ -1,136 +1,65 @@
-          variables: {
-            'nameStartsWith': model.searchController.text,
-            // fetch 30 items only, will fetch more when scrolling index is at the 3rd last item!
-            'first': 30,
-            'skip': 0,
-          },
-        ),
-        builder: (
-          QueryResult result, {
-          Future<QueryResult> Function(FetchMoreOptions)? fetchMore,
-          Future<QueryResult?> Function()? refetch,
-        }) {
-          // checking for any errors, if true fetch again!
-          if (result.hasException) {
-            final isException =
-                GraphqlExceptionResolver.encounteredExceptionOrError(
-              result.exception!,
-            );
-            print(isException);
-            if (noOfRefetch <= maxRefetch) {
-              noOfRefetch++;
-              refetch!();
-            }
-          } else {
-            // If the result is still loading!
-            if (!result.isLoading) {
-              model.organizations = OrgInfo().fromJsonToList(
-                result.data!['organizationsConnection'],
-              );
-            }
-            // return the Scroll bar widget for scrolling down the organizations.
-            return Scrollbar(
-              thumbVisibility: true,
-              interactive: true,
-              controller: model.controller,
-              // Listview is a scrollable list of widgets arranged linearly.
-              child: ListView.separated(
-                controller: model.controller,
-                padding: EdgeInsets.zero,
-                shrinkWrap: true,
-                itemCount: result.isLoading
-                    ? model.organizations.length + 1
-                    : model.organizations.length,
-                itemBuilder: (BuildContext context, int index) {
-                  // If the index is at the end of the list!
-                  if (index == model.organizations.length) {
-                    // return the ListTile showing the loading icon!
-                    return ListTile(
-                      title: Center(
-                        child: CupertinoActivityIndicator(
-                          radius: SizeConfig.screenWidth! * 0.065,
-                        ),
-                      ),
-                    );
-                  }
-                  // If the index is at the 3rd last item in the organization list.
-                  if (index == model.organizations.length - 3) {
-                    // return VisibilityDetector and fetch more items in the list to show up!
-                    return VisibilityDetector(
-                      key: const Key('OrgSelItem'),
-                      onVisibilityChanged: (VisibilityInfo info) {
-                        if (info.visibleFraction > 0) {
-                          print(model.organizations.length);
-                          model.fetchMoreHelper(
-                            fetchMore!,
-                            model.organizations,
```

Comment:
Why was all this code removed?

Reply:
Hey, I accidentally didn't add it again, was doing so for debugging purposes. Now added it back alongwith some changes